README.md

```diff
@@ -2,59 +2,71 @@
 
 The ASTT CAM software is the control and monitoring system for the AVN Satelite Teaching Telescope. The software monitors the sensors and command the telescope to move in specified direction(Az/El).
 
-The CAM Software consist of two C++ simulators(Master Node & Slave Node) which simulate the AST Telescope. However these simulators do not simulate the complete implementation of the Telescope.
+The CAM Software consist of a C++ simulator(Slave Node) which simulate the AST Telescope. However the simulator does not simulate the complete functionality of the Telescope.
 
 ## **Installation**  
 The software currently runs on:   
-* Ubuntu 20.04
+* Ubuntu machines
 
-If you are not using Ubuntu 20.04 , download docker on your machine/computer and build the software using it.
+ To install and run the simulator, download docker on your machine/computer and build the software using it.
 
 To install and run the software using docker follow this commands:
 
 * Clone this repository into your local computer.
 
     ```git clone https://github.com/AbednigoLethole/asst-cam-software.git```
   
-* Build the image.
+* Download the simulator from a drive, Here's a link below:
 
-    ```docker build -t <your image name> .```
+    ```https://drive.google.com/file/d/1JjoqoAQqZabTKoWz5sL3uhbDIZ5jXAc9/view?ts=64e315a7```
 
-* Run the docker container with admin rights.
+* start the virtual CAN Interface
 
-    ```docker run -it  --privileged <your image id> /bin/bash ```
+    ```sh startVirtualCANInterface.sh```
 
-* Once you get access to the container shell, Make shells executable.
+* Load the docker image from  the .tar file .
 
-    ```chmod +x installLely.sh startVirtualCANInterface.sh ```
+    ```docker load -i ubuntu_canopen.tar ```
 
-* Execute a shell to install Lely and start a virtual CAN Network.
+* Run the container shell that has the simulator with all its dependencies.
 
-    ```./installLely.sh ```
-    
-    ```./startVirtualCANInterface.sh ```
+    ```docker run -it --network=host ubuntu_canopen ```
+
+* Once the shell is open, move to directory that has the simulator.
+
+    ```cd simulator ```
     
+## **Running the C++ Antenna Simulator**  
+
+* The simulator is alrealdy compiled so use the following command to start it:
+
+    ```./slave ```
+
+* To restart the simulator, use the following command:
+
+    ``` reset && ./slave```
+
+## **Running the component manager**
 
-## **Running the C++ Simulators**  
+* To run the component manager install the python requirements first
 
-* move to the simulators directory.
+``` pip3 install -r requirements.txt ```
 
-    ```cd src/canopen-simulator ```
+* run the asst component manager
```

Comment:
What is asst?

Reply:
typing error it will be fixed

Reply:
Alright, I thought it stood for something that I didn't know of as it has been used even here: https://github.com/AbednigoLethole/asst-cam-software